build/utils.js

```diff
@@ -46,6 +46,14 @@ export function getRoot() {
   return path.dirname(__dirname);
 }
 
+export function safeGetBuildConfig() {
```

Comment:
Other parts of the codebase need to handle the case where getBuildConfig fails IIRC, so I opted for having a new method.